src/layers/legacy/medVtkInria/vtkImageView/vtkImageView3D.cxx

```diff
@@ -1287,3 +1289,22 @@ medVtkImageInfo* vtkImageView3D::GetMedVtkImageInfo(int layer /*= 0*/) const
 
     return imageInfo;
 }
+
+
```

Comment:
you can remove this line

Reply:
Je ne suis vraiment pas fan des fichiers qui se finisses brutalement sur une accolade, mais bon si tu veux.